L1Trigger/Phase2L1ParticleFlow/BuildFile.xml

```diff
@@ -12,6 +12,7 @@
 <use name="tensorflow"/>
 <use name="roottmva"/>
 <use name="hls"/>
+<use name="hls4mLEmulatorExtras"/>
```

Comment:
This may also need to include the name of your model in the externals here as well.